sdk/nodejs/config.ts

```diff
@@ -71,7 +72,10 @@ export class Config {
                 }
             }
         }
-        return v;
+        // SAFETY:
+        // allowedValues != null ⇒ v ∈ K[]
+        // allowedValues == null ⇒ K = string & v : string
```

Comment:
this doesn't really follow our style on comments.  We don't really try to use these sorts of mathematical invariants in comments.

Reply:
I tend to write `SAFETY` comments when performing potentially unsafe casts (`v as K`). Can remove.